civihr_employee_portal/js/ta-documents-app.js

```diff
@@ -7,11 +7,13 @@
       $urlRouterProvider.otherwise('/dashboard');
     })
     .controller('ModalController', ['$scope', '$rootScope', '$window', '$rootElement', '$log', '$uibModal',
-      'DocumentService', 'FileService', 'config', 'settings',
-      function($scope, $rootScope, $window, $rootElement, $log, $modal, DocumentService, FileService, config, settings) {
+      'DocumentService', 'FileService', 'config', 'settings', 'HR_settings',
+      function($scope, $rootScope, $window, $rootElement, $log, $modal, DocumentService, FileService, config,
+        settings, HR_settings) {
         var vm = {};
         var isContactsCached = {};
 
+        HR_settings.DATE_FORMAT = "dd/MM/yyyy";
```

Comment:
two things:
- It seems a bit weird to change the settings inside of a controller. It kind of kills the purpose of having a "global" settings var, right? Do we really need to change it in here?
- Please use single quotes to keep consistency with the rest of the code in this file